telluric/georaster.py

```diff
@@ -317,8 +326,7 @@ class GeoRaster2(WindowMethodsMixin):
     * .band_names is list of strings, order corresponding to order in .array
 
     """
-
-    def __init__(self, image=None, affine=None, crs=None, filename=None, band_names=None, nodata=0, shape=None):
+    def __init__(self, image=None, affine=None, crs=None, filename=None, band_names=None, nodata=0, shape=None, footprint=None):
```

Comment:
Are we going to protect ourselves against inconsistent input? I imagine `affine`, `crs`, `shape` and `footprint` can overdetermine the image.